Resources/Prototypes/Recipes/Construction/weapons.yml

```
@@ -242,3 +242,15 @@
+
+# Assmos - Metal Hydrogen
+- type: construction
+  name: metal hydrogen axe
+  id: MetalHydrogenAxeConstruct
+  graph: MetalHydrogenAxeConstruct
+  startNode: start
+  targetNode: icon
+  category: construction-category-weapons
+  description: A lightweight crowbar with an extreme sharp fire axe head attached. It trades its heft as a weapon by making it easier to carry around when holstered to suits without having to sacrifice your backpack.
+  icon: { sprite: _Goobstation/Objects/Weapons/Melee/metal_hydrogen_axe.rsi, state: icon }
+  objectType: Item
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)